tutorials/cloudsdk-e2e-test/cloudsdk-e2e-test.md

```diff
@@ -16,13 +16,11 @@ This tutorial assumes you are already familiar with the basics of the SAP Cloud
 To follow this tutorial please download and extract an [example project](https://github.com/SAP/cloud-s4-sdk-book/archive/tutorials/testing-start.zip) based on the SAP Cloud SDK archetype.
 
 
-
-## Intro
 > ## We migrate tutorials to our [documentation](https://sap.github.io/cloud-sdk/)
```

Comment:
Do not header formatting to emphasize text. \*\*We migrate...\*\* would be correct in this case.